rlog.go

```diff
@@ -416,6 +571,14 @@ func basicLog(logLevel int, traceLevel int, format string, prefixAddition string
 	}
 }
 
+// Flush syncs the content of the logfile, which might still be in a memory
+// buffer, to storage. This should rarely be necessary during normal operation.
+func Flush() {
```

Comment:
Who is supposed to call this?
What happens to this unsynced data when a program panics or `os.Exit()`'s?